bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -249,8 +251,12 @@ const BaseContainer = withTracker(() => {
     userSubscriptionHandler = Meteor.subscribe('users', credentials, mappedUser.isModerator, subscriptionErrorHandler);
     breakoutRoomSubscriptionHandler = Meteor.subscribe('breakouts', credentials, mappedUser.isModerator, subscriptionErrorHandler);
     meetingModeratorSubscriptionHandler = Meteor.subscribe('meetings', credentials, mappedUser.isModerator, subscriptionErrorHandler);
+
+    userInfoSubscriptionhandler = Meteor.subscribe('users-infos', credentials, subscriptionErrorHandler);
```

Comment:
Why do we need this handler here? From my understanding we only need this if there's a change between moderator and viewer published data. In that case we'd have more conditions in the Publisher and we'd be passing `mappedUser.isModerator` here

Reply:
Also adjust case here - this is likely breaking: `userInfoSubscriptionhandler` vs `userInfoSubscriptionHandler`